### PR TITLE
Add TomTom GO

### PR DESCRIPTION
**Pull request by Laura7089**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -12300,6 +12300,13 @@
         ]
     },
 
+    {
+        "name": "TomTom GO",
+        "url": "https://help.tomtom.com/hc/en-gb/requests/new?ticket_form_id=360000697820",
+        "difficulty": "medium",
+        "notes": "Fill in the form to automatically open a support ticket."
```
Comment (tupaschoal) [inline _data/sites.json:12307]: If it requires support to be involved, and thus manual human labour, we classify it as hard

File: _data/sites.json [hunk 2]
```diff
@@ -12300,6 +12300,16 @@
         ]
     },
 
+    {
+        "name": "TomTom GO",
+        "url": "https://help.tomtom.com/hc/en-gb/requests/new?ticket_form_id=360000697820",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:12306]:
```suggestion
        "difficulty": "hard",
```

**Comment by Laura7089** [comment]
Oh, my bad - must have messed up a rebase

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
> Oh, my bad - must have messed up a rebase, sorry

Don't worry, it happens :)

Thanks for the contribution!

**Pull request merged**